mobile-workspace/latest/using/index.md

```diff
@@ -133,3 +133,51 @@ You can move files within the Mobile Workspace.
 
     * The scan will be saved where you navigated.
     * If you do not wish to keep the scan, exit without saving. A dialog box will appear to confirm you want to discard the scan, press **Discard**. Press **Cancel** to return to save your scan.
+
+## Tasks
+
+You can view, filter, complete, add comments, and add attachments to tasks while logged into the Mobile Workspace.
```

Comment:
Add attachment is not part of Goring release. User can view the  attachments.
Add attachment will be the part of Cholsey release.

Reply:
I've removed adding attachments part of the general tasks description.